lib/holidays.rb

```diff
@@ -9,44 +9,6 @@
 require 'holidays/errors'
 require 'holidays/load_all_definitions'
 
-# == Region options
```

Comment:
Fair point. I'm not as anti-comment as some of my coworkers (who refuse to allow *any* of them, ever). I bet that you and I are probably pretty close on what we would find 'acceptable'.

In this case I'll merge this and think about the best way to document for future developers.

Thanks!